areas/room.cpp

```diff
@@ -717,6 +718,8 @@ void displayRoom(Player* player, const BaseRoom* room, int magicShowHidden) {
                     oStr << "(desc)";
                 if(ext->flagIsSet(X_NEEDS_FLY))
                     oStr << "(fly)";
+            } else if(hasBeenUsedBy && ext->isDiscoverable()) {
```

Comment:
Since I'm proposing removing the `hasBeenUsedBy` and just using canSee, can change this to just isDiscoverable; because it'll only be visible if they can see it (ie: is discoverable, and they've used it)

Reply:
I'd be fine with the following:

(*) [and they have D-I]
(fly)
(h) [Same as you have it now, for players]

Thoughts @ocelot134269 ?

Reply:
That is fine by me. If players can see the tags, we might think about chaning * to "inv" or "invis" possibly. Or just "i"

Reply:
Cool we'll just leave it alone then.